manifests/p/PaulPacifico/ShutterEncoder/18.6/PaulPacifico.ShutterEncoder.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: PaulPacifico.ShutterEncoder
+PackageVersion: "18.6"
+PackageLocale: en-US
+Publisher: Paul Pacifico
+PackageName: Shutter Encoder
+License: Proprietary
+Copyright: Paul Pacifico 2013-2024
+ShortDescription: Shutter Encoder

```

Comment:
Please provide a better description of what `Shutter Encoder` is, does, or is used for